.circleci/config.yml

```diff
@@ -510,3 +509,5 @@ jobs:
           path: test/integration/render-tests
       - store_artifacts:
           path: "test/integration/render-tests/index.html"
+
+# VS Code Extension Version: 1.5.1
```

Comment:
Should this stay or is it a leftover?

Reply:
Looks like [developers have complained about it](https://discuss.circleci.com/t/please-do-not-modify-the-config-file-with-a-comment-indicating-the-vs-code-extension-version/46403) and it will be removed in a future version, so let's cut it!